### DISCUSSION TITLE
Avoid SSL in deploy

### DISCUSSION THREAD

**Discussion by xavism** [vercel/vercel#4560] [Help] [answered]
I'm deploying an app that sends info to an API. 
I cannot add an SSL to my api so the connection app (https) -> api (http) is not possible.
Is it possible to serve the app only through HTTP?

Thanks!

**Comment by xavism** [comment]
Duplicated #1745

**Comment by coetry** [comment, marked as answer]
Hi @xavism we don't currently have the option to serve traffic over HTTP because we want to establish secure defaults and drive our customers towards success. Serving traffic over HTTPS is an industry-standard that we wouldn't want to steer away from.
